Lab1b/syntax.in.txt

```diff
@@ -22,9 +13,6 @@ ARRAYDECL = GENERICTYPE "[" NUMBER "]"
 
 TYPE = GENERICTYPE | ARRAYDECL
 
-<<<<<<< HEAD
-CMPDSTMT = "{" STMT "}"
-
 STMT = (SIMPLESTMT | STRUCTSTMT) {STMT}
```

Comment:
Why do you need to group SIMPLESTMT and STRUCTSTMT?

Reply:
`SIMPLESTMT` is for statements like assign and iostmt, which need to end with `;`, whereas `STRUCTSTMT` is for complex statement, like if, while etc that do not end in `;`